packages/cli/src/utils/generate.js

```diff
@@ -110,7 +110,21 @@ async function getNuxt (args, cmd) {
   config.buildDir = (config.static && config.static.cacheDir) || path.resolve(config.rootDir, 'node_modules/.cache/nuxt')
   config.build = config.build || {}
   config.build.transpile = config.build.transpile || []
-  config.build.transpile.push(config.buildDir)
+  if (config.static && config.static.cacheDir) {
+    if (
+      !config.build.transpile.some(path =>
```

Comment:
Transpile also supports functions ([example](https://github.com/nuxt/http/blob/v0.5.0/lib/module.js#L106). Simply puting `.cache/nuxt` without warning could be enough :)

Reply:
Okay - and we can always specify in the docs if/when there are docs for this option :stuck_out_tongue: